### PR TITLE
Increment the versions and adjust the capitilization

### PR DESCRIPTION
**Pull request by nickvergessen**

### CODE REVIEW

File: apps/files_sharing/appinfo/info.xml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0"?>
 <info>
 	<id>files_sharing</id>
-	<name>Share Files</name>
+	<name>Files sharing</name>
```
Comment (blizzz) [inline apps/files_sharing/appinfo/info.xml]: Using singular seems more usual: https://en.wikipedia.org/wiki/File_sharing

**Comment by MorrisJobke** [comment]
Beside the comment of Arthur this looks good 👍

File: apps/files_sharing/appinfo/info.xml (hunk 1 above)
Comment (nickvergessen) [inline apps/files_sharing/appinfo/info.xml]: Fixed

**Comment by MorrisJobke** [comment]
👍

**Comment by rullzer** [comment]
I fixed the intergration tests (admin audit is no longer enabled by default, trivial patch).
:+1:

**Comment by MorrisJobke** [comment]
@rullzer Could I ask you to create the backport PR? Thanks :)

**Comment by rullzer** [comment]
@MorrisJobke already done by @nickvergessen https://github.com/nextcloud/server/pull/738

**Comment by MorrisJobke** [comment]
> @MorrisJobke already done by @nickvergessen #738

@rullzer When this is created we could also drop the backport-request label. Makes it then easier to find the not yet backported PRs ;)

**Pull request merged**